config/src/token_remaps.rs

```diff
@@ -84,11 +102,31 @@ pub fn setup_token_remaps(remap_file: Option<String>, chain: Chain) -> Result<()
     match ADDR_DECIMALS_MAP.get() {
         Some(_) => {
             warn!("Token decimals map already set, cannot override");
-            Ok(())
         },
         None => ADDR_DECIMALS_MAP
             .set(decimals_map)
-            .map_err(raw_err_str!("Failed to set token decimals map: {:?}")),
+            .map_err(raw_err_str!("Failed to set token decimals map: {:?}"))?,
+    };
+
+    // Set up exchange support map, be sure to add coinbase/kraken support for dummy
+    // USD token
+    let mut exchange_support_map = map.to_exchange_support_map();
+    // Insert exchange support for the dummy USD token
+    exchange_support_map.insert(
+        usd_ticker.clone(),
+        [(Exchange::Coinbase, usd_ticker.clone()), (Exchange::Kraken, usd_ticker.clone())]
+            .into_iter()
+            .collect(),
+    );
+
+    match EXCHANGE_SUPPORT_MAP.get() {
+        Some(_) => {
+            warn!("Exchange support map already set, cannot override");
+            Ok(())
+        },
```

Comment:
this should probably just panic like the others do, a warning would be too easy to look over, so no need to match here.

Reply:
Mm, we also handle double-setting of the `TOKEN_REMAP` and `ADDR_DECIMALS_MAP` in this way, but I think panicking is appropriate in this case. So I'll update those to panic if the `OnceLock` is already set as well

Reply:
for posterity, calling `set` on the `OnceLock` only fails if it is already set, so I remove all the matches and just make a call to `set`, propagating the error instead of panicking